proto/streamlit/proto/Element.proto

```diff
@@ -108,8 +109,8 @@ message Element {
     VegaLiteChart vega_lite_chart = 10;
     Video video = 14;
     Heading heading = 47;
-
-    // Next ID: 48
+    Code code = 49;
```

Comment:
I think this should be 48, because that's the next available ID. Doing a check I don't see 48 being used. (Feel free to adjust the comment to be more descriptive).